.github/ISSUE_TEMPLATE/member_build_cache_issue.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 'Internal: Issue for the `build-cache` team'
```

Comment:
Let's rename the file to reflect the title.

Reply:
Done